src/topology/vector_bundle.lean

```diff
@@ -78,6 +90,13 @@ namespace topological_vector_bundle
 
 variables {R F E}
 
+/-- Natural identification as `topological_vector_prebundle.trivialization`. -/
+def trivialization.to_pretrivialization (e : trivialization R F E) :
+  topological_vector_prebundle.trivialization R F E := { ..e }
+
+instance : has_coe (trivialization R F E) (topological_vector_prebundle.trivialization R F E) :=
```

Comment:
You shouldn't register coercions too eagerly, because the system doesn't always work very well (and won't work the same way in Lean 4). If you expect to use the coercion often, it's fine to have it. If you expect to use it just two or three times, it's probably not worth it. In this specific case, I don't think it's worth it because when we have a genuine vector bundle, why would we consider it as a prebundle?

Reply:
Yeah sure I agree